SU2_CFD/src/solvers/CAdjNSSolver.cpp

```diff
@@ -773,8 +773,8 @@ void CAdjNSSolver::Viscous_Sensitivity(CGeometry *geometry, CSolver **solver_con
             Enthalpy = solver_container[FLOW_SOL]->GetNodes()->GetEnthalpy(iPoint);
 
             /*--- Turbulent kinetic energy ---*/
-
-            if ((config->GetKind_Turb_Model() == TURB_MODEL::SST) || (config->GetKind_Turb_Model() == TURB_MODEL::SST_SUST))
+            /* Nijso says: This does not seem to be consistent with the primal treatment. */
+            if (config->GetKind_Turb_Model() == TURB_MODEL::SST) 
               val_turb_ke = solver_container[TURB_SOL]->GetNodes()->GetSolution(iPoint,0);
             else
               val_turb_ke = 0.0;
```

Comment:
This has an impact on if we take into account the divergence term of the tke,  so I think we only should set a nonzero val_turb_ke in case of the original 1994, 2003, or 1994v and 2003v model

Reply:
so this seems to be consistent with the current sst implementation, but we have to modify this condition when we fix the sst1994 model. 
I think for V2003m (current implementation) we should put val_turb_ke=0.0 since we drop all tke divergence terms?